src/common/hex-strings.ts

```diff
@@ -76,15 +96,19 @@ export const padHexString = (str: string, length: number): string => {
   }
 }
 
-export const getLen = (pos: { start; end }) => (pos.end - pos.start) * 2
-
-export const encodeHex = (val: any, len: number) =>
-  remove0x(BigNumber.from(val).toHexString()).padStart(len, '0')
-
-export const toVerifiedBytes = (val: string, len: number) => {
+// TODO: Review these three functions, see if we can simplify and use other functions instead.
+export const toVerifiedBytes = (val: string, len: number): string => {
```

Comment:
@tynes maybe we can find an easier way to do these operations using the above functions.